diff --git a/chrome/common/importer/profile_import.typemap b/chrome/common/importer/profile_import.typemap
index 6283f2bf6871a10f710694772b5da0bc9b70c2ad..78968b5fba8693185c8e517d3d08b140b4a49d70 100644
--- a/chrome/common/importer/profile_import.typemap
+++ b/chrome/common/importer/profile_import.typemap
@@ -4,6 +4,10 @@
 
 mojom = "//chrome/common/importer/profile_import.mojom"
 public_headers = [
+  "//onevn/common/importer/onevn_ledger.h",
+  "//onevn/common/importer/onevn_stats.h",
+  "//onevn/common/importer/onevn_referral.h",
+  "//onevn/common/importer/imported_browser_window.h",
   "//chrome/common/importer/imported_bookmark_entry.h",
   "//chrome/common/importer/importer_autofill_form_data_entry.h",
   "//chrome/common/importer/importer_data_types.h",
@@ -13,6 +17,7 @@ public_headers = [
 traits_headers =
     [ "//chrome/common/importer/profile_import_process_param_traits.h" ]
 deps = [
+  "//onevn/common",
   "//chrome/common",
   "//components/favicon_base",
   "//ipc",
@@ -26,4 +31,11 @@ type_mappings = [
   "chrome.mojom.SearchEngineInfo=::importer::SearchEngineInfo",
   "chrome.mojom.SourceProfile=::importer::SourceProfile",
   "chrome.mojom.ImportItem=::importer::ImportItem",
+  "chrome.mojom.OneVNStats=::OneVNStats",
+  "chrome.mojom.OneVNLedger=::OneVNLedger",
+  "chrome.mojom.OneVNReferral=::OneVNReferral",
+  "chrome.mojom.ImportedBrowserTab=::ImportedBrowserTab",
+  "chrome.mojom.ImportedBrowserWindow=::ImportedBrowserWindow",
+  "chrome.mojom.ImportedWindowState=::ImportedWindowState",
+  "chrome.mojom.SessionStoreSettings=::SessionStoreSettings"
 ]
