diff --git a/extensions/browser/extension_event_histogram_value.h b/extensions/browser/extension_event_histogram_value.h
index 124a05e2a09c40af404840140a4403fef85e40e7..21f36ab151b11b35837f1294f10c4a215ddbd073 100644
--- a/extensions/browser/extension_event_histogram_value.h
+++ b/extensions/browser/extension_event_histogram_value.h
@@ -456,6 +456,20 @@ enum HistogramValue {
   ACCESSIBILITY_PRIVATE_ON_ANNOUNCE_FOR_ACCESSIBILITY = 435,
   MIME_HANDLER_PRIVATE_SAVE = 436,
   RUNTIME_ON_CONNECT_NATIVE = 437,
+
+  ONEVN_START = 600,
+  ONEVN_AD_BLOCKED,
+  ONEVN_WALLET_CREATED,
+  ONEVN_ON_WALLET_PROPERTIES,
+  ONEVN_ON_PUBLISHER_DATA,
+  ONEVN_ON_CURRENT_REPORT,
+  ONEVN_ON_ONEVN_THEME_TYPE_CHANGED,
+  ONEVN_REWARDS_NOTIFICATION_ADDED,
+  ONEVN_REWARDS_NOTIFICATION_DELETED,
+  ONEVN_REWARDS_ALL_NOTIFICATIONS_DELETED,
+  ONEVN_REWARDS_GET_NOTIFICATION,
+  ONEVN_REWARDS_GET_ALL_NOTIFICATIONS,
+  ONEVN_WALLET_FAILED,
   // Last entry: Add new entries above, then run:
   // python tools/metrics/histograms/update_extension_histograms.py
   ENUM_BOUNDARY
